vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -149,11 +149,14 @@ $global:imagesToPull += @(
     # CNS (Container Networking Service) Owner: evanbaker
     "mcr.microsoft.com/containernetworking/azure-cns:v1.4.52",
     "mcr.microsoft.com/containernetworking/azure-cns:v1.5.38",
+    "mcr.microsoft.com/containernetworking/azure-cns:v1.5.40",
     "mcr.microsoft.com/containernetworking/azure-cns:v1.6.13",
 
     # CNI installer for azure-vnet. Owner: evanbaker
     "mcr.microsoft.com/containernetworking/azure-cni:v1.4.56",
+    "mcr.microsoft.com/containernetworking/azure-cni:v1.4.58",
```

Comment:
I don't think we have enough VHD space to have a new CNI version - is there any way we can remove .56?

Reply:
there never is 🥲 I'll re-evaluate this now that #5441 is merged